integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDropTableCommand.scala

```diff
@@ -143,7 +143,6 @@ case class CarbonDropTableCommand(
       OperationListenerBus.getInstance.fireEvent(dropTablePostEvent, operationContext)
     } catch {
       case ex: NoSuchTableException =>
-        LOGGER.error(ex.getLocalizedMessage, ex)
```

Comment:
Logging this event is necessary. Instead of printing the whole exception stack we can just print the message as a warning.

Reply:
Done